### ISSUE TITLE
[Suggestion] Better air charm

### ISSUE THREAD

**Issue by ghost** [PrinceOfAmber/Cyclic#598]
It says in JEI that the [air charm](/PrinceOfAmber/Cyclic/wiki/Air-Charm) allows you to walk on air, but it’s basically just flight—the only difference being that it’s slower than creative flight, it can’t be toggled by double-tapping space, and you slowly descend whilst in the air.

I personally think that it should allow you to walk “on” air—but _not_ fly. For instance, you would be able to walk from platform A to platform B as if there were a bridge there, or you could descend to the ground, but you wouldn’t be able to **ascend**.

The reason for my opinion is that this is a really cheap way of gaining flight at the moment. Nether wart, iron, and bones are easy to get, and a single diamond isn’t too hard to obtain early-game either.

The recipe could, of course, be more expensive, but there are better methods of flying in many different mods. I’d prefer if it were a more unique relatively cheap early-game/early mid-game (if you will) way of “flight.”

**Comment by PrinceOfAmber** [comment]
Thanks for your patience. I have thought about this of an on a few times.  Some of my thoughts are below. Despite all this, i might remove or completely overahul the item in future, such as making it a pair of special boots.  (This was inspired initially by an artifiact armor I saw in somebodys Project ozone 2 playthrough.)

"a more unique relatively cheap early-game/early mid-game (if you will) way of “flight.”", Yes this is the intention.  I feel like it is cheap early game, and its a weaker version of flight.
- In the past ive thought about making it more powerful/faster
- Yes its a cheap but worse version of flight, by design.  
- your title says "better" but in your text you mention not allowing vertical ascent, so thats a worse version
- even as it is now, every single stream or video ive watched, people either skip over this for Real Flying (simple gems, angel ring) or they get this, use it for a while, and then move on.  No indication its overpowered
- If you have diamonds and netherwart, you are pretty much at endgame anyway
- recipe and durability can be changed with things like craft tweaker.


.. Dont get me wrong, i love the feedback. and the fact that this item doesnt get used much does say something.  I think youre onto something with making two versions, one stronger than what it currently is, and one weaker

I have added this to my project board under "overhaul feature" category for future planning, closing the issue for now.
